.circleci/config.yml

```diff
@@ -229,7 +229,7 @@ jobs:
           name: "Run Unit Tests"
           command: |
             export PATH=~/adapter-deps/install/bin:${PATH}
-            make unittestrel VELOX_ENABLE_ADAPTERS=ON AWSSDK_ROOT_DIR=~/adapter-deps/install NUM_THREADS=16
+            make unittestonly BUILD_DIR=release NUM_THREADS=16
```

Comment:
This change makes sense, but why don't we simply invoke `ctest` directly to avoid any confusion?

```
ctest --test-dir=_build/release -j16 -VV --output-on-failure
```

We're passing 2/4 flags explicitly already. We'd probably want to track down why we have some tests excluded, to be fair, but it's not clear why we have these long lived exclusions.

Reply:
I agree that invoking `ctest` directly is best.

Reply:
But as you mentioned, let's first resolve the exclusions. That way, we don't have to clean up multiple targets.

Reply:
Do you want me to move ahead with committing this change?

Reply:
Yea, hold on. I can take a look at the excluded tests first.